RetailSuiteAccounting/Integration/01_RunAccountingWizard.md

```diff
@@ -0,0 +1,109 @@
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+### Start wizard
+
+*Accounting*
+
+![Accounting wizard](../../Assets/Screenshots/RetailSuiteAccounting/AccountingWizard/AccountingWizard.png "[Accounting wizard]")
+
+Click the [START] button.   
+The configuration process starts. The *Fiscal years* wizard window is displayed.
+
+> [Info] The progress in the wizard is displayed on the top left corner in the accounting wizard window.
+
+
+### Fiscal years
+
+In the first step you have to configure the fiscal years.
+
+![Fiscal years](../../Assets/Screenshots/RetailSuiteAccounting/AccountingWizard/FiscalYears.png "[Fiscal years]")
+
+ 1. Enter a description for the fiscal year in the *Description* field. The system shows the current year by default.  
+
+   > [Info] It is possible to enter digits, letters or combinations of characters. The number of characters is limited to 10.
+
+ 2.  Enter the starting and finishing date of your fiscal year in the *From - To* fields. The fiscal year shown by default is the current calendar year.
+
+ > [Info] The fiscal year must consist of 12 months. However, it does not have to be a calendar year, for example from 1st April to 31st March.
+
+ 3. Select the base currency in the *Currency* drop-down list. The drop-down list displays all currencies configured in the system. However, the accounting module works with the base currency only.
+
+
+ 4. Select the appropriate option (**No/Yes**) in the *Cash accounting* drop-down list.
+
+ > [Info] The cash accounting refers to a method where receipts are recorded during the period they are received, and expenses are recorded in the period in which they are actually paid. This contrasts accrual accounting, where revenue is recognized when earned rather than when collected, and expenses are recognized when incurred rather than when paid.
+
+ > [Warning] This option has fundamental implications for the system automatic recording of taxes. In case of doubt, please check with your tax advisor.
+
+ 5. Select the appropriate option (**No/Fiscal year for planning purposes only**) in the *Planning FY* drop-down list.   
+
+ > [Info] Note that a fiscal year can be set up for planning purposes only. However, the fiscal year for planning purposes will not be recognized as an actual fiscal year by the system, and therefore the automatic functions in the interaction with other modules will not apply.
+
+ 6. Click [SAVE & NEW].  
+  The new fiscal year is created.    
+A small pop-up window and a green check mark confirm that the new fiscal year has been saved. The newly created fiscal year is also displayed in the list.
```

Comment:
```suggestion
  The new fiscal year is created.  A pop-up window and a green check mark confirm that the new fiscal year has been saved. The newly created fiscal year is also displayed in the list.
```
Wo wird das grüne Häkchen angezeigt? Bild vom Pop-Up oder Fenster? In welcher Liste wird das Wirtschaftsjahr angezeigt?